### PR TITLE
Clarify upgrade procedure

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

**Comment by ochriste** [comment]
thank you very much! 
please merge at will

File: doc/integrator/install_application.rst [hunk 1]
```diff
@@ -294,6 +294,11 @@ configuration:
 
 .. note::
 
+   For technical simplification the file should be named ``<instanceid>.mk``
+   in other world ``<instanceid>`` should have the same value as ``<user>``.
```
Comment (kalbermattenm) [inline doc/integrator/install_application.rst]: world -> word

File: c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl [hunk 2]
```diff
@@ -805,6 +805,13 @@ $(APACHE_CONF_DIR)/$(INSTANCE_ID).conf:
 UPGRADE_MAKE_FILE ?= $(INSTANCE_ID).mk
 UPGRADE_ARGS ?=
 
+$(UPGRADE_MAKE_FILE):
+	@echo "The file '$(UPGRADE_MAKE_FILE)' can't be found,"
+	@echo "this probably mean that you Makefile isn't named '<instanceid>.mk'."
+	@echo "To solve this you can rename your Makefile or add in your makefile"
```
Comment (asaunier) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: Please be consistent, choose Makefile or makefile and use it all the times :P

File: c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl [hunk 3]
```diff
@@ -805,6 +805,13 @@ $(APACHE_CONF_DIR)/$(INSTANCE_ID).conf:
...
+	@echo "`UPGRADE_MAKE_FILE = <your_makfile_name>`."
```
Comment (asaunier) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: your_makfile_name => your_makefile_name

File: doc/integrator/install_application.rst (hunk 1 above)
Comment (asaunier) [inline doc/integrator/install_application.rst]: world => words (I guess :P)

Comment (asaunier) [inline doc/integrator/install_application.rst]: @kalbermattenm You're too fast for me!

Comment (kalbermattenm) [inline doc/integrator/install_application.rst]: Yeah... I just updated my comment... sorry :-/

**Comment by sbrunner** [comment]
Thanks @ochriste, @kalbermattenm and @asaunier :-)

**Pull request merged**